api-authentication/server/server.ts

```diff
@@ -80,7 +80,7 @@ app.post('/api/auth/sign-in', async (req, res, next) => {
       userId,
       username,
     };
-    const token = jwt.sign(payload, 'snitchesgetstiches');
+    const token = jwt.sign(payload, process.env.TOKEN_SECRET as string);
```

Comment:
The type assertion works. The better solution, though, would be to verify that TOKEN_SECRET is defined. The reason you can't pass it directly to `jwt.sign` is because `jwt.sign` cannot take `undefined` as its second parameter. TypeScript knows that, and also knows that `.env` variables can possibly be undefined. Hover over TOKEN_SECRET to see its type and you will see that it is `string | undefined`. It's the `undefined` that TS is complaining about.